config/localization/en/app.json

```diff
@@ -954,6 +954,9 @@
       "SLOW": "Slow",
       "SLOW_WITHDRAW_DESCRIPTION": "Slow withdraws are a two step process. This step initiates your withdraw request and will be processed on Layer 2. Once your funds are ready (can take up to {HOURS} hours), you will receive a notification to withdraw your funds to your wallet. You are responsible for the withdraw transaction gas fees.",
       "SLOW_WITHDRAW_CONFIRMATION": "This {STYLED_SLOW} withdraw may take up to {HOURS} hours before it is available to withdraw. You will be responsible for the withdraw transaction gas fees. If you need your funds sooner, try a {STYLED_FAST} withdraw.",
+      "SUB_HEADER_PRE_TOOLTIP": "USDC withdrawals to",
+      "SUB_HEADER_TOOLTIP": "select chains",
+      "SUB_HEADER_POST_TOOLTIP": "have the lowest fees. Other withdrawals may have additional third-party fees",
```

Comment:
hmm why aren't these in the tooltips.json [file](https://github.com/dydxprotocol/v4-localization/blob/main/config/localization/en/tooltips.json) isntead?

Reply:
same comment about splitting internationalization*

Reply:
![image](https://github.com/dydxprotocol/v4-localization/assets/37092291/1bfbf05f-81f1-4960-9393-3d1b799b2c51)
so these are referring to the subheader text here. so the `SUB_HEADER_TOOLTIP` is sub header text that is wrapped in a `WithTooltip` component. naming could probably be better here.

would params work here? since we need to inject a withtooltip?

Reply:
hmm, in this case should the text that appears on the modal itself be the `TITLE` while the `BODY` refers to the text that appears inside the actual tooltip popup?